src/Razor/src/Microsoft.VisualStudio.RazorExtension/source.extension.vsixmanifest

```diff
@@ -53,6 +53,7 @@
     <Asset Type="Microsoft.VisualStudio.Assembly" Path="Microsoft.Extensions.Logging.Abstractions.dll" />
     <Asset Type="Microsoft.VisualStudio.Assembly" Path="System.Reactive.dll" />
     <Asset Type="Microsoft.VisualStudio.Assembly" Path="Microsoft.AspNetCore.Razor.LanguageServer.dll" />
+    <Asset Type="Microsoft.VisualStudio.Assembly" Path="Microsoft.AspNetCore.Razor.LanguageSupport.dll" />
```

Comment:
@DustinCampbell It seems weird to me that we were able to merge without this before but when I tried to run things locally I ran into problems without it, do you have any ideas?